src/store/student-store.js

```diff
@@ -18,13 +18,8 @@ class StudentStore extends Store{
 const store = new StudentStore();
 
 store.registerHandler('STUDENT_FOUND', data => {
-    student = {
-        //NOTE: this data is tentative
-        id : data.id,
-        name: data.name,
-        items: data.items,
-        hasOverdueItem: store.hasOverdueItems(data.items)
-    };
+    student = data;
```

Comment:
TLDR; I hid away the attributes in the client and you need to look for attributes in the api documents for consus to prevent attributes missing from server and client because we forgot to explicitly define them.